### PR TITLE
Fixed some function deprecations, a bogus stackpointer clobber and a …

### PR DESCRIPTION
**Pull request by sfjuocekr**
…redundant __packed reference the compiler complained about it not doing anything.

### CODE REVIEW

File: radio/src/targets/horus/lcd_driver.cpp [hunk 1]
```diff
@@ -142,6 +142,7 @@ static void LCD_NRSTConfig(void)
 }
 
 // TODO delay function
+#if !defined(RADIO_T18)
```
Comment (3djc) [inline radio/src/targets/horus/lcd_driver.cpp:145]: What's that for ?

Comment (sfjuocekr) [reply radio/src/targets/horus/lcd_driver.cpp:145]: Because that function is only used if that radio is not defined if you look down at line 158

Comment (bsongis) [reply radio/src/targets/horus/lcd_driver.cpp:145]: it would be nice to use delay_us anyhow!

Comment (sfjuocekr) [reply radio/src/targets/horus/lcd_driver.cpp:145]: I left the original code mostly as-is and just fixed some compiler complaints.

Looking at the code that would be a huge number to put into delay_us, the person who put it there probably had a reason for it and sadly it is not commented either.

I'm guessing the datasheet of the display/controller would have the reason documented.

**Pull request closed without merging**